src/minuteman_lashup_vip_listener.erl

```diff
@@ -255,13 +257,17 @@ rewrite_name({{tcp, {name, {Name, FWName}}, PortNum}, BEs}) ->
 rewrite_name(Else) ->
     Else.
 
-retry_spartan(State) ->
-    push_state_to_spartan(State).
+retry_spartan(State0) ->
+    State1 = push_state_to_spartan(State0),
+    State1.
 
-push_state_to_spartan(State) ->
+push_state_to_spartan(State0) ->
     ZoneNames = ?ZONE_NAMES,
-    Zones = lists:map(fun(ZoneName) -> zone(ZoneName, State) end, ZoneNames),
-    push_zones(Zones).
+    Zones = lists:map(fun(ZoneName) -> zone(ZoneName, State0) end, ZoneNames),
+    push_zones(Zones),
+    Timer = timer:send_after(?RPC_RETRY_TIME, retry_spartan),
```

Comment:
I don't think we need to store the `Timer` here, the timer will just send a message to `self()` after `?RPC_RETRY_TIME`